app/models/submission.rb

```diff
@@ -197,8 +197,9 @@ def make_remark_result
     end
 
     original_result.marks.each do |mark|
-      remark_result.marks.create(mark.attributes.merge(
-                                   result: remark, created_at: Time.zone.now))
+      remark_result.marks.create(result: remark, created_at: Time.zone.now, 
+                                  markable_id: extra_mark.markable_id, 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.